bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sonnen"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sonnen Thing Type -->
+	<thing-type id="battery">
+		<label>sonnen Binding Thing</label>
+		<description>Thing for sonnen battery connection</description>
+
+		<channels>
+			<channel id="channelBatteryCharging" typeId="isBatteryCharging"/>
+			<channel id="channelBatteryDischarging" typeId="isBatteryDischarging"/>
+			<channel id="channelConsumption" typeId="consumption"/>
+			<channel id="channelGridFeedIn" typeId="gridFeedIn"/>
+			<channel id="channelSolarProduction" typeId="solarProduction"/>
+			<channel id="channelBatteryLevel" typeId="batteryLevel"/>
+			<channel id="channelFlowConsumptionBattery" typeId="isFlowConsumptionBattery"/>
+			<channel id="channelFlowConsumptionGrid" typeId="isFlowConsumptionGrid"/>
+			<channel id="channelFlowConsumptionProduction" typeId="isFlowConsumptionProduction"/>
+			<channel id="channelFlowGridBattery" typeId="isFlowGridBattery"/>
+			<channel id="channelFlowProductionBattery" typeId="isFlowProductionBattery"/>
+			<channel id="channelFlowProductionGrid" typeId="isFlowProductionGrid"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostIP" type="text" required="true">
+				<context>network-address</context>
+				<label>IP Address</label>
+				<description>Please add the IP Address of your sonnen battery</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="0" max="1000">
+				<label>Refresh Interval</label>
+				<description>How often the sonnen battery in seconds should schedule a refresh after a channel is linked to an item.
+					Valid input is 0 - 999..</description>
```

Comment:
I believe this is more correct in English: How often in seconds the sonnen...
If 1000 is not a valid value, use `max=999`

Reply:
ok